### PR TITLE
Code quality/combine consecutive issets combine consecutive unsets

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
Adding two additional php--cs-fixer rules

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: core/ajax/share.php [hunk 1]
```diff
@@ -271,10 +271,10 @@ function usersInGroup($gid, $search = '', $limit = -1, $offset = 0) {
...
 			break;
 		case 'getItem':
-			if (isset($_GET['itemType'])
-				&& isset($_GET['itemSource'])
-				&& isset($_GET['checkReshare'])
-				&& isset($_GET['checkShares'])) {
+			if (isset($_GET['itemType'], $_GET['itemSource'], $_GET['checkReshare'], $_GET['checkShares'])
+				 
+				 
+				 ) {
```
Comment (mmattel) [inline core/ajax/share.php]: maybe to remove the empty lines...

**Comment by mmattel** [comment]
https://github.com/owncloud/documentation/issues/4064

File: lib/private/Server.php [hunk 2]
```diff
@@ -684,7 +684,7 @@ public function __construct($webRoot, \OC\Config $config) {
 					'server' => $_SERVER,
 					'env' => $_ENV,
 					'cookies' => $_COOKIE,
-					'method' => (isset($_SERVER) && isset($_SERVER['REQUEST_METHOD']))
+					'method' => (isset($_SERVER, $_SERVER['REQUEST_METHOD'])  )
```
Comment (phil-davis) [inline lib/private/Server.php]: And it left space between the closing brackets ``) )``
I suspect that when we add more checks, space in places like that will fail the code quality tests.

**Comment by phil-davis** [comment]
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3339 explains why ``combine_consecutive_issets`` leaves space behind.
The suggested solution is to also use the rule ``no_spaces_inside_parenthesis``

**Review by phil-davis — APPROVED** [review]
@DeepDiver1975 I suggest we merge this, and immediately after merge #31273 which fixes up a bunch of white-space stuff.

That should annoy everyone with an open PR quite enough for 1 day ;)

**Comment by phil-davis** [comment]
Effective backport is included in #31453

**Pull request merged**